packages/KStepper/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@kongponents/kstepper",
+  "version": "6.36.3",
```

Comment:
You'll need to update the version here and in the file below

Reply:
Fixed